lang/pt.po

```diff
@@ -0,0 +1,39 @@
+msgid ""
+msgstr ""
+"Language: pt\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "msg_help_usage"
+msgstr "Uso: vrms-rpm [opções]"
+msgid "msg_help_option_ascii"    
+msgstr "    Mostra rms arte ASCCI quando não é encontrado nenhum pacote livre,\n"
+       "    ou quando os pacotes não livres representam mais do que 10%% do total."
+msgid "msg_help_option_explain"
+msgstr "    Quando lista os pacotes, mostra a licença para justificar\n"
+       "    a classificação de livre / não livre."
+msgid "msg_help_option_help"
+msgstr "    Mostra esta ajuda e termina."
+msgid "msg_help_option_list"
+msgstr "    Para além de mostrar um resumo do numero de pacotes livres e não livres,\n"
+       "    lista-os por nome. O valor por defeito é 'nonfree'."
+msgid "msg_help_option_version"
+msgstr "    Mostrar informação de versão e terminar."
+msgid "msg_list_noargument"
+msgstr "%s: a opção --list requer um argumento"
+msgid "msg_list_badargument"
+msgstr "%s: argumento para a opção --list tem que ser um de 'none', 'free', 'nonfree' ou 'all'"
+msgid "msg_unknown_option"
+msgstr "%s: opção desconhecida '%s'"
+msgid "msg_total_free"
+msgstr "%d pacotes livres"
+msgid "msg_total_nonfree"
+msgstr "%d pacotes livres"
+msgid "msg_rms_happy_ascii"
+msgstr "Só pacotes livres - rms estaria orgulhoso!                           rms está orgulhoso."
```

Comment:
This particular line shouldn't extend past 79 characters. Remove some spaces between the two texts.